incubator/nginx-iap/pr123/apple.yaml

```diff
@@ -0,0 +1,26 @@
+kind: Pod
```

Comment:
are the files in `pr123` folder just examples?
how they are going to be deployed?
maybe just move to examples in README?